Makefile

```diff
@@ -94,3 +94,16 @@ site-publish-only:
 	(cd ./_site && git add .)
 	(cd ./_site && git commit -am 'update site')
 	(cd ./_site && git push git@github.com:Travix-International/frint gh-pages --force)
+
+##
+# Usage stats
+#
+define list_usage_in_source
```

Comment:
I think setting additional ESLint rule to disallow full imports of `rxjs` and `lodash` would be beneficial (https://eslint.org/docs/rules/no-restricted-imports)

Reply:
awesome idea! I have implemented the ESLint check in most recent commit.

I would still keep these Makefile commands though. they help us quickly find out which modules we currently need. and the output of these commands is the primary source of information for creating `frint-config`.

Reply:
Nice. Of course, Makefile commands are also useful